doc/dadbod.txt

```diff
@@ -81,6 +81,29 @@ Handling these parameters is up to the individual adapters.  By default,
 unrecognized parameters are ignored.  Pass Boolean parameters as the literal
 strings "true" and "false" (e.g., `?ssl=true`).
 
+                                                *dadbod-bigquery*
+bigquery ~
+>
+    bigquery:[?<globalflag>=<value>][&<globalflag>=<value>][...]
+<
+This adapter uses the `bq` command line tool provided by the Google Cloud SDK.
+Please ensure this is installed first as per: https://cloud.google.com/sdk/docs/install.
+
+Please note that `bq` accepts both *global* flags and *command* flags for the
+`bq query` subcommand, however *only* global flags are currently supported as
+query parameters by dadbod. To specify subcommand defaults, please use your
+local `~/.biqgqueryrc`.
+
+The query *execution* project and default dataset may be specified in the address hostname
```

Comment:
Asterisks are used to denote help tags. Don't use them for emphasis/italics. (I don't think any of this is especially important to emphasis at any rate.)

Last line needs to be rewrapped.